### PR TITLE
Searcher Icons 3

### PR DESCRIPTION
**Pull request by s9ferech**
### Pull Request Description

This PR provides the remaining icons that have been planned for the searcher. The following screenshot shows all icons. The highlighted icons have been added or changed in this PR.

### Checklist
Please include the following checklist in your PR:

- [ ] The `CHANGELOG.md` was updated with the changes introduced in this PR.
- [ ] The documentation has been updated if necessary.
- [x] All code conforms to the [Rust](https://github.com/enso-org/enso/blob/main/docs/style-guide/rust.md) style guide.
- [ ] All code has automatic tests where possible.
- [ ] All code has been profiled where possible.
- [x] All code has been manually tested in the IDE.
- [x] All code has been manually tested in the "debug/interface" scene.
- [x] All code has been manually tested by the PR owner against our [test scenarios](https://docs.google.com/spreadsheets/d/1RatJDM_f9_3bvYhl3Bpq2d8SyKgtVdrV1RkGxPU17c8/edit?ts=5faa7049#gid=0).
- [ ] All code has been manually tested by at least one reviewer against our [test scenarios](https://docs.google.com/spreadsheets/d/1RatJDM_f9_3bvYhl3Bpq2d8SyKgtVdrV1RkGxPU17c8/edit?ts=5faa7049#gid=0).

[ci no changelog needed]

### CODE REVIEW

File: src/rust/ide/view/src/searcher/icons.rs [hunk 1]
```diff
@@ -63,6 +63,16 @@ fn grid(stroke_width:f32,cell_size:f32) -> AnyShape {
...
 
+/// A path following the given points. All joints and endpoints are rounded.
+fn path(width:f32, points:&[(f32,f32)]) -> AnyShape {
+    let points     = points.iter().map(|(x,y)| (x.px(),y.px()));
+    let mut result : AnyShape = EmptyShape().into();
+    for (start,end) in points.clone().zip(points.skip(1)) {
+        result = (result + Segment(start,end,width.px())).into();
+    }
+    result
+}
```
Comment (wdanilo) [inline src/rust/ide/view/src/searcher/icons.rs]: That is so cool implementation!

Comment (wdanilo) [reply src/rust/ide/view/src/searcher/icons.rs]: It should be refactored somewhere where we define shapes in general. It is very useful!

**Review by wdanilo — APPROVED** [review]
Amaing work! The `path` implementation is brilliant.

File: src/rust/ide/view/src/searcher/icons.rs (hunk 1 above)
Comment (s9ferech) [reply src/rust/ide/view/src/searcher/icons.rs]: @wdanilo I am not sure where the definition should go. Should it be inside `display::shape::compound` or `display::shape::primitive`?

Comment (wdanilo) [reply src/rust/ide/view/src/searcher/icons.rs]: lets move it to `display::shape::compound::path` module (file named path.rs).

Comment (s9ferech) [reply src/rust/ide/view/src/searcher/icons.rs]: Done

**Pull request merged**